zstacklib/ansible/zstacklib.py

```
@@ -1589,14 +1589,19 @@ def get_remote_host_info_obj(host_post_info):
 
     ret = result['contacted'][host]
     if 'ansible_facts' in ret:
+        # trying get cpu info
+        command = 'cat /proc/cpuinfo | grep -i -E "model name|vendor"'
+        (status, cpu_info_output) = run_remote_command(command,
```

Ruff (0.14.4)

Unpacked variable `status` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)